### ISSUE TITLE
Doesn't work with other BG layer

### ISSUE THREAD

**Issue by bobvanaubel** [ddwht/sketch-dynamic-button#12]
I found an issue where the plugin doesn't to work when you have another layer inside an artboard called 'BG'. When I selected the text and pressed 'CMD-J', it placed the text in the left top corner of my artboard with the original BG layer underneath it.

![screen shot 2015-03-10 at 17 54 45](https://cloud.githubusercontent.com/assets/5420002/6580115/a7129212-c74e-11e4-8e1c-f99513e65690.png)

In my case it was a folder with a mask inside of it. Don't know if this also happens with other objects called 'BG'.

**Comment by ddwht** [comment]
Yes, plugin tries to found pair BG Layer and Text Layer with paddings. And after that it reads paddings and apply it on the BG. If you have own BG layer plugin applied paddings to it.
